Install/Chroot

```diff
@@ -36,15 +36,15 @@ Almost all modern Linux distributions support network configuration and have dow
 
 So it's usually not a problem. Though if you don't have (or you know that you won't have Internet-connection inplace for some reason) such ability, you can put the desired chroot image on your USB-key or CD/DVD disc.
 
-You can download the appropriate chroot image from our [[Download|downloads]] page; and we will use the one based on **0.60-stable** grimoire in the example below.
+You can download the appropriate chroot image from our [[Download|downloads]] page; and we will use the one based on **0.63-test** grimoire in the example below. While others are listed, I have found that they are too old and either to not provide a stable system to start with, or break during the update process noted towards the end of this guide. Also note, while the 0.63-test version is specifically mentioned, the datetime stamp may be different on the official download page as updates to the 0.63-test chroot image are made.

```

Comment:
s/to not/do not/

Reply:
This typo has been corrected in a minor commit. This should be pulled in along with the next pull request as I get further along with some of the other recommendations.